irony-cdb-json.el

```diff
@@ -100,6 +111,30 @@ it could easily be used with helm, for instance, by enabling
     (irony-cdb-json--save-project-alist)
     (irony-cdb-autosetup-compile-options)))
 
+(defun irony-cdb-json--last-mod (file)
+  (nth 5 (file-attributes file)))
```

Comment:
I actually tries this but got an error and got away from it again. I like this solution far better.